lib/utils/package_utils.rb

```diff
@@ -5,21 +5,22 @@
 module MeasureRepositoryServiceTestKit
   # Utility functions in support of the $package test group
   module PackageUtils
-    def related_artifacts_present?(bundle)
+    def related_artifacts_present?(bundle, valuesets)
```

Comment:
Super nitpick here, but I think for boolean args it can be clearer to name them something like `shouldCheckValuesets` or something just to really hammer home that it's a boolean arg. I'd also make this arg an optional one that is defaulted to `false`